index.bs

```diff
@@ -4590,3 +4763,14 @@ Possible mitigations:
     presence from the reporting origin. Compared to the previous mitigation, the
     proxy server could itself handle the [=event-level report/trigger priority=]
     functionality, at the cost of increased complexity in the proxy.
+
+## Attribution Scope ## {#attribution-scope}
+
+*This section is non-normative.*
+
+Abusive actors can exploit the attribution scope feature by registering "ghost navigation sources" instead of event sources: During source registration, they can redirect to themselves and register [=attribution source/attribution scope limit=] - 1 navigations. This allows them to choose more complex configurations (more bits) for their ghost registrations.
```

Comment:
We may want to clarify that self-redirects aren't the only way to achieve this. The other approaches are:

1. Multiple background requests.
2. A foreground request and a background request.